### PR TITLE
Fix #5120 Missing Text Margins In Topic Revision Cards in Large Screens

### LINKED ISSUE #5120: [BUG]: Missing Text Margins In Topic Revision Cards in Large Screens
### Describe the bug

Unlike in small screens, the tablet UI of revision cards does not have any margins/padding.

| Small screen | Tablet|
| ------ | ------ |
|Screenshot_1691303581| Screenshot_1690255965|

### Steps To Reproduce

Install Oppia on a real tablet or emulator such as a Nexus 9 device.
On a profile home screen, click on any topic.
On the topic list screen, switch to the Revision tab and observe that there is no margin.

### Expected Behavior

There should be some text margin.

### Screenshots/Videos

_No response_

### What device/emulator are you using?

_No response_

### Which Android version is your device/emulator running?

_No response_

### Which version of the Oppia Android app are you using?

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by pingu-73**
## Explanation

Fix #5120 [Missing Text Margins In Topic Revision Cards in Large Screens](https://github.com/oppia/oppia-android/issues/5120) 

Added layout_marginStart and layout_marginEnd in oppia-android/app/src/main/res/layout-sw600dp/revision_card_fragment.xml
Screenshot 2023-08-09 at 4 52 46 PM

Absence of marginStart and marginEnd caused missing text margins on Large Screens. 

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

## Screenshots
**before fix:**
padding and margin

**after fix:**
ratiocorrect

### CODE REVIEW

File: .idea/codeStyles/Project.xml
Comment (adhiamboperes) [inline .idea/codeStyles/Project.xml:1]: Please remove this file

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @pingu-73!


Please read the PR description and make sure you have addressed the items on the checklist that I have unchecked.

File: .idea/codeStyles/Project.xml
Comment (pingu-73) [reply .idea/codeStyles/Project.xml:1]: do I need to remove .idea folder or project.xml file?

Comment (MohitGupta121) [reply .idea/codeStyles/Project.xml:1]: Yes @pingu-73, you have to follow all conditions mentioned in Essential Checklist of your PR description.

Comment (pingu-73) [reply .idea/codeStyles/Project.xml:1]: @MohitGupta121 is it fine?

Comment (MohitGupta121) [reply .idea/codeStyles/Project.xml:1]: No you have to remove unnecessary files

Comment (pingu-73) [reply .idea/codeStyles/Project.xml:1]: @MohitGupta121 @adhiamboperes please have a look at it now

**Comment by pingu-73** [comment]
@MohitGupta121 can you please approve checks

**Comment by MohitGupta121** [comment]
@pingu-73 It's okay, after your code is approved the maintainer will approve the checks also. Don't worry.

**Review by adhiamboperes — APPROVED** [review]
Thanks @pingu-73! 

Congratulations! Your changes LGTM.

**Pull request merged**
